src/nvim/testdir/test_gf.vim

```diff
@@ -75,8 +77,10 @@ func Test_gF()
   edit Xfile1
   call setline(1, ['one', 'Xfile:4', 'three'])
   exe "normal 2G\<C-W>F"
-  call assert_equal('Xfile', bufname('%'))
-  call assert_equal(4, getcurpos()[1])
+ "FIXME
+ "call assert_equal('Xfile', bufname('%'))
+ "FIXME
+ "call assert_equal(4, getcurpos()[1])
```

Comment:
This is a somewhat important feature of vim. Perhaps `path_is_url` can check if the part immediately after the colon is a number? If deno is using `deno:<number>`, well...